checkout-v3/starter/seamless-view.md

```diff
@@ -168,8 +168,8 @@ Content-Type: application/json
 //to open the Payment Menu
 window.payex.hostedView.checkout({
     container: {
-        checkin: "checkin",
-        paymentMenu: "paymentMenu",
+        checkin: "checkin-container",
+        paymentMenu: "paymentMenu-container",
     },
     culture: 'nb-No',
 }).open("paymentmenu");
```

Comment:
You need to change the HTML example below as well. I can't make a code review suggestion for it since it's not changed in this PR. It needs to be changed from:

```html
            <div id="checkin"></div>
            <div id="payment-menu"></div>
```

to:

```html
            <div id="checkin-container"></div>
            <div id="payment-menu-container"></div>
```